js/piwik.js

```diff
@@ -3290,7 +3290,8 @@ if (typeof window.Piwik !== 'object') {
                     (msToExpire ? ';expires=' + expiryDate.toGMTString() : '') +
                     ';path=' + (path || '/') +
                     (domain ? ';domain=' + domain : '') +
-                    (isSecure ? ';secure' : '');
+                    (isSecure ? ';secure' : '') +
+                    (isSecure ? ';SameSite=None' : ';SameSite=Lax');
```

Comment:
Actually that might maybe cause problems in some specific browser versions that do not accept cookies with `SameSite=None` or treat them as `SameSite=Strict` instead. There's a list of incompatible browser versions. See https://www.chromium.org/updates/same-site/incompatible-clients